tests/components/watergate/test_init.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for the Watergate integration init module."""
+
+from unittest.mock import patch
+
+from homeassistant.components.valve import ValveState
+from homeassistant.components.watergate.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from . import init_integration
+from .const import MOCK_WEBHOOK_ID
+
+from tests.common import ANY
+from tests.typing import ClientSessionGenerator
+
+
+async def test_async_setup_entry(
+    hass: HomeAssistant, mock_entry, mock_watergate_client
```

Comment:
```suggestion
    hass: HomeAssistant, mock_entry, mock_setup_entry: AsyncMock ,mock_watergate_client,
```
Here it should also be ok, plus please add type hints for all parameters in tests where it's missing

Reply:
Well, test is called test_async_setup_entry, and I am checking if webhooks are setup correctly so I cannot path the setup_entry.

Reply:
Right brain lag. Don't forget adding the type hints though